How to use the go library ?

Hello,

I'm trying to use the _s4_ go library to encrypt and decrypt bytes.
But when using `import "github.com/simonfrey/s4"` as written in the README, I get a strange error : 

> go: finding github.com/simonfrey/s4 latest
go: downloading github.com/simonfrey/s4 v0.0.0-20210502093622-aa95753ec8f6
go: extracting github.com/simonfrey/s4 v0.0.0-20210502093622-aa95753ec8f6
go: test imports
        github.com/simonfrey/s4: github.com/simonfrey/s4@v0.0.0-20210502093622-aa95753ec8f6: parsing go.mod:
        module declares its path as: s4
                but was required as: github.com/simonfrey/s4

I've tried the rewrite directive in my _go.mod_ but it's only making things worse. How to correctly import this library ?
Thanks !

Tested on my repo, I think this PR shoudl fix this for good !

I was getting this error without this : 
`../go/pkg/mod/github.com/simonfrey/s4@v0.0.0-20210726193420-57fa43c35b94/s4.go:7:2: package s4/crypto is not in GOROOT (/usr/lib/go-1.17/src/s4/crypto)`

--- a/s4.go
+++ b/s4.go
@@ -4,7 +4,7 @@ import (
 	"bytes"
 	"github.com/hashicorp/vault/shamir"
 	"github.com/pkg/errors"
-	"s4/crypto"
+	"github.com/simonfrey/s4/crypto"
 )
 
 var splitCode []byte = []byte("\n*=_=_=_=*\n\n")

--- a/wasm/wasm.go
+++ b/wasm/wasm.go
@@ -3,8 +3,8 @@ package main
 import (
 	"encoding/base64"
 	"fmt"
+	"github.com/simonfrey/s4"
 	"regexp"
-	"s4"
 	"strconv"
 	"strings"
 	"syscall/js"

